testing/bdd/features/deduplicator/6Sty3Dup6Reg3DupForeignNamespace.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" ?>
+<tt:tt ebuttp:sequenceIdentifier="ReSequenced1" ebuttp:sequenceNumber="12" ttp:cellResolution="32 15" ttp:clockMode="local" ttp:timeBase="clock" xml:lang="en-GB" xmlns:ns1="urn:namespace1" xmlns:ns2="urn:namespace2" xmlns:ebuttp="urn:ebu:tt:parameters" xmlns:ebutts="urn:ebu:tt:style" xmlns:tt="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:xml="http://www.w3.org/XML/1998/namespace">
+  <tt:head>
+    <tt:styling>
+      <!--  2, 3 and 4 should be merged -->
```

Comment:
Interesting case! Not sure what the handling of foreign namespaces attributes should be but treating them as the test suggests is reasonable.